### DISCUSSION TITLE
786

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#958] [Comments]
# 786

최근 신혼 여행을 가면서 비행기에서 와이프가 잠들때마다 오늘만 사는 기사 라는 웹 소설을 봤다.주인공은 예전에 본 영화 엣지 오브 투모로우 처럼 죽으면 그 날로 다시 되살아나는 저주에 걸려있는 채로 매일 하루를 반복하며 목표로 하는 기사가 되기 위해 노력하는 이야기이다.소울풍 작가님의 과거 작품을 좋아하기도 했고, 한산이가 작가님과 함께 하시는 유튜브를 통해 새 작품 소식을 듣고 봐야지 봐야지 하다가 짬이 되어 이제야 보기 시작했다.소설 45화에 기억에 남는 장면이 있는데,적진에 고립되는 높은 난이도의 상황에 처하게 되어 계속 죽게 된다.매일 똑같은 상황에서 죽고, 되살아나고, 노력하고 다시 시도하고의 하루를 반복하다가 결국 여러 운과 그간의 노력이 합쳐져 통해 아슬아슬하게 그 상황을 돌파하게 된다.그리..

https://jojoldu.tistory.com/786

**Comment by wanderer-s** [comment]
향로님도 마성의 중대장에게 빠지셨군요

**Comment by zusdn90** [comment]
매번 좋은글 감사합니다.ㅎㅎ

아래 문장에 어색한 부분이 있어서 코멘트 남겨봅니다.

결국 여러 운과 그간의 노력이 합쳐져 통해 아슬아슬하게 그 상황을 돌파하게 된다.

**Reply by jojoldu** [reply]
앗 ㅋㅋㅋㅋㅋㅋ 감사합니다! ㅋㅋㅋㅋ
